src/db/schemas/user.ts

```diff
@@ -1,6 +1,7 @@
 import * as bcrypt from 'bcrypt-nodejs';
 import * as mongoose from 'mongoose';
 
+import { emailActions } from '../../config/config';
```

Comment:
You could rename ../../config/config.ts to ../../config/index.ts so it'd be easier to require it by doing just 'from '../../config'. But keep it like this. This is ok too.